cocos/renderer/CCTexture2D.h

```diff
@@ -380,6 +382,9 @@ class CC_DLL Texture2D : public Ref
     Texture2D* getAlphaTexture() const;
 
     bool getAlphaTextureName() const;
+    
+    void setupProgram(float globalZOrder);
+    
```

Comment:
Does this function need to be public?

Reply:
I think it's fine to be public, we can call it when needed.